contact.htm

```diff
@@ -2,43 +2,45 @@
-            </div>
-            <div class="col-12 col-sm-8 col-md-8 col-lg-8">
-                <h3 class="text-bold">How to get in contact with us!</h3>
-                <p>Our large userbase is spread across the following chat platforms:</p>
-                <ul>
-                    <li>
-                        <a href="https://t.me/gridcoin">Gridcoin's Telegram channel</a>
-                    </li>
-                    <li>
-                        <a href="https://join.slack.com/t/teamgridcoin/shared_invite/enQtMjk2NTI4MzAwMzg0LTE4N2I3ZWZjYWJlZGM1Zjg3MTUyMDhiN2M5NmRmZTA2NDA0ZmY1ZTFmOGM3ZGU2YTBkOTdhNTk2ZjkzMGZkODY">Gridcoin's Slack channel</a>
-                    </li>
-                    <li>
-                        <a href="https://discord.gg/jf9XX4a">Gridcoin's Discord channel</a>
-                    </li>
-                    <li>
-                        <a href="https://www.reddit.com/r/gridcoin">/r/gridcoin subreddit</a>
-                    </li>
-                    <li>
-                        <a href="https://steemit.com/created/gridcoin">#Gridcoin Steemit tag</a>
-                    </li>
-                </ul>
-            </div>
-            <div class="col-12 col-sm-12 col-md-12 col-lg-12">
-                <p>If you're looking to report security concerns:</p>
-                <ul>
-                    <li>See the official <a href="https://github.com/gridcoin-community/Gridcoin-Research/blob/master/SECURITY.md">Gridcoin Security Disclosure Document</a>.</li>
-                    <li>Email "<a href="mailto:contact@gridcoin.us">contact@gridcoin.us</a>" to get the team's attention.</li>
-                    <li>Jump into the <a href="https://grcinvite.herokuapp.com/">Slack channel</a> or the <a href="https://t.me/gridcoin">Telegram channel</a> and ask for an administrator.</li>
-                </ul>
-                <p>If you have non-critical Gridcoin client issues/bugs you wish to raise awareness of, please <a href="https://github.com/gridcoin-community/Gridcoin-Research/issues">submit an issue on the GitHub repo</a>.</p>
-                <p>If you're interested in collaborating towards Gridcoin's future development, don't hesitate to join Slack/Telegram/Discord/GitHub to begin working with others on your ideas.</p>
-            </div>
-        </div>
-    </div>
+<section id="section_resources_intro" class="text-center">
+  <h1 class="echo-header">
+      <span class="echo" aria-hidden="true">Contact Us</span>
+      Contact <strong>Us</strong>
+  </h1>
+  <img src="/assets/img/ic-contact-us.svg" alt="How to get in contact with us">
+</section>
+
+<section id="section_contact_us" class="container">
+  <div class="row">
+      <div class="col-12">
+          <p>Our large userbase is spread across the following chat platforms:</p>
+          <ul>
+              <li>
+                  <a target="_blank" rel="nofollow" href="https://t.me/gridcoin">Gridcoin's Telegram channel</a>
```

Comment:
is changing the target to `_blank` a documented change?  What is the reasoning?

Reply:
The reason is that I want people to not to leave the website. But I agree I went a little bit mad :-) Prob it is not necessary as the links are pointing to the own resources. Will roll those changes back.